CHANGELOG.md

```diff
@@ -27,6 +27,9 @@ Line wrap the file at 100 chars.                                              Th
 ### Added
 - Added possibility to filter locations by provider in the desktop app.
 - Add WireGuard over TCP CLI option for all relays.
+- New GUI environment variable `DISABLE_VERSION_CHECK` has been added. If set to `1`, latest
```

Comment:
Changelog entries should begin with a verb, e.g. add, fix, increase or force. As specified further up in the changelog.
> Entries should have the imperative form, just like commit messages. Start each entry with words like
add, fix, increase, force etc.. Not added, fixed, increased, forced etc.

Just changing the first sentence to this would solve it:
> Add GUI environment variable `DISABLE_VERSION_CHECK`